### PR TITLE
Onion slicess grammar fix

### LINKED ISSUE #70004: Double S in onion slicess
Reporting client version: 514.1583

## Round ID:
[190665](https://scrubby.melonmesa.com/round/190665)

## Reproduction:
image

### PR DESCRIPTION
**Pull request by Salex08**
## About The Pull Request

Fixes https://github.com/tgstation/tgstation/issues/70004
Grammar fix

## Changelog

:cl:
spellcheck: corrects plural spelling when slicing objects
/:cl:

### CODE REVIEW

**Review by ShizCalev — CHANGES_REQUESTED** [review]
`gender = PLURAL`

**Comment by Salex08** [comment]
> `gender = PLURAL`

ok

File: code/game/atoms.dm [hunk 1]
```diff
@@ -1490,7 +1490,7 @@
 				created_atom.pixel_y += rand(-8,8)
 			SEND_SIGNAL(created_atom, COMSIG_ATOM_CREATEDBY_PROCESSING, src, chosen_option)
 			created_atom.OnCreatedFromProcessing(user, process_item, chosen_option, src)
-			to_chat(user, span_notice("You manage to create [chosen_option[TOOL_PROCESSING_AMOUNT]] [initial(atom_to_create.name)]\s from [src]."))
+			to_chat(user, span_notice("You manage to create [chosen_option[TOOL_PROCESSING_AMOUNT]] [initial(atom_to_create.gender) == PLURAL ? "[initial(atom_to_create.name)]" : "[initial(atom_to_create.name)]\s"] from [src]."))
```
Comment (ShizCalev) [inline code/game/atoms.dm:1493]: Sorry for not being more concise. 
This should be automatic with` \s`, I meant you need to set the gender of the onions.

Comment (Salex08) [reply code/game/atoms.dm:1493]: ahhh I thought you tried to imply that they are supposed to be plural

File: code/modules/hydroponics/grown/onion.dm [hunk 2]
```diff
@@ -21,6 +21,7 @@
 	seed = /obj/item/seeds/onion
 	name = "onion"
 	desc = "Nothing to cry over."
+	gender = PLURAL
```
Comment (ShizCalev) [inline code/modules/hydroponics/grown/onion.dm:24]: this isn't onion slices.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/hydroponics/grown/onion.dm (hunk 2 above)
Comment (Salex08) [reply code/modules/hydroponics/grown/onion.dm]: ok

File: code/game/atoms.dm (hunk 1 above)
Comment (ShizCalev) [reply code/game/atoms.dm]: Oh crap my bad. I could swear I checked & they didn't have the gender var set to plural. This was the right fix lmao

Comment (Salex08) [reply code/game/atoms.dm]: i see alright

Comment (Salex08) [reply code/game/atoms.dm:1493]: done

**Comment by dragomagol** [comment]
While I assume this fixes an existing issue, it is still just several single onion slices that are created when you slice the onion.

**Pull request merged**